docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -170,3 +170,61 @@ Result:
     ]]                                                                                                                   │
 └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
+## seriesDecomposeAnomaliesDetection
```

Comment:
So the new function detects outliers, right? And for that it uses series decomposition, correct?

To stay consistent with the existing [naming](https://clickhouse.com/docs/en/sql-reference/functions/time-series-functions#seriesoutliersdetecttukey) of time series functions, I suggest name `seriesOutliersDetectDecompose`.

Reply:
`Outlier detection` is different from `Anomalies Detection`. Outlier detection uses standardized methods like IQR or variance to find outliers. It does not consider the seasonality or trends in data which are expected in a time series data. 
Anomaly detection through decomposition takes into account the seasonality and trends in time series data. 

So, it could be named `seriesAnomaliesDetectDecompose`?